src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -104,7 +105,7 @@ object ConductRPlugin extends AutoPlugin {
 
     val projectImageVersion = if (hasRpLicense.value) Some(LatestConductRVersion) else None
     val bundlePorts =
-      (BundleKeys.endpoints in Bundle).?.map(_.getOrElse(Map.empty)).all(filter).value
+      BundleKeys.endpoints.?.map(_.getOrElse(Map.empty)).all(filter).value
```

Comment:
This looks fine, just curious about the consequences of the change. ReactiveMaps may be a good test for this.